packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -68,13 +68,20 @@ const defaultProps = {
   selectedDate: undefined,
 };
 
+const contextTypes = {
+  /* eslint-disable consistent-return */
+  intl: (context) => {
```

Comment:
Do we like this pattern vs. using injectIntl to add an intl prop?  I don't think this needs to hold us up on this PR, but I'm more curious if we prefer one vs. the other outright or have situations where one makes sense over the other.